lib/model-helper.js

```diff
@@ -226,6 +226,12 @@ var modelHelper = module.exports = {
 
       if (arrayItem) {
         if(typeof arrayItem === 'object') {
+          if (arrayItem.enum) {
```

Comment:
@shelbys This is not performing validation. It is to convert from LDL to swagger config.